homeassistant/components/lutron/light.py

```diff
@@ -25,14 +31,38 @@ async def async_setup_entry(
     Adds dimmers from the Main Repeater associated with the config_entry as
     light entities.
     """
+    ent_reg = er.async_get(hass)
     entry_data: LutronData = hass.data[DOMAIN][config_entry.entry_id]
-    async_add_entities(
-        [
-            LutronLight(area_name, device, entry_data.client)
-            for area_name, device in entry_data.lights
-        ],
-        True,
-    )
+
+    for area_name, device in entry_data.lights:
+        if device.type == "CEILING_FAN_TYPE":
+            # If this is a fan check to see if this entity already exists. If not, do not create a new one.
+            entity_id = f"light.{slugify(f'{area_name} {device.name}')}"
```

Comment:
The user may have renamed the entity_id. Look up the entity entry from the unique id instead.